index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <link href="https://fonts.googleapis.com/css?family=Inter:400,600,500,800" rel="stylesheet">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./style/style.css">
+    <title>Portfolio</title>
+</head>
+<body>
```

Comment:
- [ ] [REQUIRED] As stated in the project requirements, please ensure you stick to the design as much as possible. In your case, please include the background image decorations.

### Your design
![one](https://user-images.githubusercontent.com/63964646/148427304-81897f7d-09cb-40e8-8188-559d89f50aef.PNG)

### Original Design
![two](https://user-images.githubusercontent.com/63964646/148427323-1a4fb92d-fa9f-4e64-a80a-33a831b08235.PNG)

Reply:
@lily-coder the background image is included and shown in the README file. Maybe it's a slow loading page

This is a screenshot of it
<img width="338" alt="mobile" src="https://user-images.githubusercontent.com/40362530/148432421-36a4f09b-6551-418e-afae-1ac3a200ddb3.png">